src/main/java/uk/gov/hmcts/reform/civil/validation/interfaces/ParticularsOfClaimValidator.java

```diff
@@ -3,18 +3,33 @@
 import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
 import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
 import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.model.CaseData;
 import uk.gov.hmcts.reform.civil.model.ServedDocumentFiles;
 
 import static java.util.Optional.ofNullable;
 
 public interface ParticularsOfClaimValidator {
 
+    private ServedDocumentFiles getServedDocumentFiles(CaseData caseData) {
```

Comment:
Maybe we should use `CallbackParams` as an argument here instead of `CaseData`, so we don't have to call the method with `callbackParams.getCaseData()` everywhere